src/component/splits/mod.rs

```diff
@@ -425,61 +425,79 @@ impl Component {
         let mut settings = SettingsDescription::with_fields(vec![
-            Field::new("Background".into(), self.settings.background.into()),
             Field::new(
-                "Total Splits".into(),
+                "Background".into(),
+                "The background shown behind the component. You can choose for the colors to be alternating. In that case each row alternates between the two colors chosen.".into(),
+                self.settings.background.into(),
+            ),
+            Field::new(
+                "Total Rows".into(),
+                "The total number of rows of segments to show in the list. If set to 0, all the segments are shown. If set to a number lower than the total amount of segments, only a certain window of all the segments is shown. This window can scroll up or down.".into(),
                 Value::UInt(self.settings.visual_split_count as _),
             ),
             Field::new(
-                "Upcoming Splits".into(),
+                "Upcoming Segments".into(),
+                "If there's more segments than rows that are shown, the window showing the segments automatically scrolls up and down when the current segment changes. This number determines the minimum number of future segments to be shown in this scrolling window when it automatically scrolls.".into(),
                 Value::UInt(self.settings.split_preview_count as _),
             ),
             Field::new(
                 "Show Thin Separators".into(),
+                "Specifies whether thin separators should be shown between the individual segments shown by the component.".into(),
                 self.settings.show_thin_separators.into(),
             ),
             Field::new(
                 "Show Separator Before Last Split".into(),
+                "If the last segment is to always be shown, this determines whether to show a more pronounced separator in front of the last segment, if it is not directly adjacent to the segment shown right before it in the scrolling window.".into(),
                 self.settings.separator_last_split.into(),
             ),
             Field::new(
                 "Always Show Last Split".into(),
+                "If not every segment is shown in the scrolling window of segments, then this option determines whether the final segment is always to be shown, as it contains valuable information about the total duration of the chosen comparison, which is often the runner's Personal Best.".into(),
                 self.settings.always_show_last_split.into(),
             ),
             Field::new(
-                "Fill with Blank Space if Not Enough Splits".into(),
+                "Fill with Blank Space".into(),
+                "If there's not enough segments to fill the list, this option allows filling the remaining rows with blank space in order to maintain the number of total rows specified. Otherwise the number of total rows shown is reduced to the actual amount of segments.".into(),
```

Comment:
```suggestion
                "If there's not enough segments to fill the list, this option allows filling the remaining rows with blank space in order to always show the number of total rows specified in the settings. Otherwise, the number of total rows shown is reduced to the actual number of segments.".into(),
```